hi-ml-histopathology/src/histopathology/models/deepmil.py

```diff
@@ -308,7 +308,15 @@ def update_results_with_data_specific_info(self, batch: dict, results: dict) ->
                         ResultsKey.TILE_ID: batch[TilesDataset.TILE_ID_COLUMN],
                         ResultsKey.IMAGE_PATH: batch[TilesDataset.PATH_COLUMN]})
 
-        if (TilesDataset.TILE_X_COLUMN in batch.keys()) and (TilesDataset.TILE_Y_COLUMN in batch.keys()):
+        if (TileKey.TILE_TOP in batch.keys()) and (TileKey.TILE_LEFT in batch.keys()) \
```

Comment:
[Optional] A couple of less verbose alternatives (for here and in `metrics_utils.py`):
```python
# Test if set of keys is subset of all batch keys
if {TileKey.TILE_TOP, TileKey.TILE_LEFT, TileKey.TILE_RIGHT, TileKey.TILE_BOTTOM} <= batch.keys():
```
or more explicitly:
```python
if all(key in batch for key in [TileKey.TILE_TOP, TileKey.TILE_LEFT, TileKey.TILE_RIGHT, TileKey.TILE_BOTTOM]):
```

Reply:
There's also `a.issubset(b)`, which is, I think, more explicit than `a <= b` for sets.